temporalio/contrib/openai_agents/temporal_openai_agents.py

```diff
@@ -133,3 +142,121 @@ def stream_response(
+
+    This plugin provides seamless integration between the OpenAI Agents SDK and
+    Temporal workflows. It automatically configures the necessary interceptors,
+    activities, and data converters to enable OpenAI agents to run within
+    Temporal workflows with proper tracing and model execution.
+
+    The plugin:
+    1. Configures the Pydantic data converter for type-safe serialization
+    2. Sets up tracing interceptors for OpenAI agent interactions
+    3. Registers model execution activities
+    4. Manages the OpenAI agent runtime overrides during worker execution
+
+    Args:
+        model_params: Configuration parameters for Temporal activity execution
+            of model calls. If None, default parameters will be used.
+        model_provider: Optional model provider for custom model implementations.
+            Useful for testing or custom model integrations.
+
+    Example:
+        >>> from temporalio.client import Client
+        >>> from temporalio.worker import Worker
+        >>> from temporalio.contrib.openai_agents import OpenAIAgentsPlugin, ModelActivityParameters
+        >>> from datetime import timedelta
+        >>>
+        >>> # Configure model parameters
+        >>> model_params = ModelActivityParameters(
+        ...     start_to_close_timeout=timedelta(seconds=30),
+        ...     retry_policy=RetryPolicy(maximum_attempts=3)
+        ... )
+        >>>
+        >>> # Create plugin
+        >>> plugin = OpenAIAgentsPlugin(model_params=model_params)
+        >>>
+        >>> # Use with client and worker
+        >>> client = await Client.connect(
+        ...     "localhost:7233",
+        ...     plugins=[plugin]
+        ... )
+        >>> worker = Worker(
+        ...     client,
+        ...     task_queue="my-task-queue",
+        ...     workflows=[MyWorkflow],
+        ... )
+    """
+
+    def __init__(
+        self,
+        model_params: Optional[ModelActivityParameters] = None,
+        model_provider: Optional[ModelProvider] = None,
+    ) -> None:
+        """Initialize the OpenAI agents plugin.
+
+        Args:
+            model_params: Configuration parameters for Temporal activity execution
+                of model calls. If None, default parameters will be used.
+            model_provider: Optional model provider for custom model implementations.
+                Useful for testing or custom model integrations.
+        """
+        self._model_params = model_params
+        self._model_provider = model_provider
+
+    def configure_client(self, config: ClientConfig) -> ClientConfig:
+        """Configure the Temporal client for OpenAI agents integration.
+
+        This method sets up the Pydantic data converter to enable proper
+        serialization of OpenAI agent objects and responses.
+
+        Args:
+            config: The client configuration to modify.
+
+        Returns:
+            The modified client configuration.
+        """
```

Comment:
No need to change anything, but for overrides, especially ones users never call (and never should) you don't _have_ to have docstrings, though it's harmless.